docs/source/torch/auto_deploy/auto-deploy.md

```diff
@@ -28,7 +28,7 @@ AutoDeploy provides an alternative method for deploying models using the LLM API
 AutoDeploy is included with the TRT-LLM installation.
 
 ```bash
-sudo apt-get -y install libopenmpi-dev && pip3 install --upgrade pip setuptools && pip3 install tensorrt_llm
+sudo apt-get -y install libopenmpi-dev libzmq3-dev && pip3 install --upgrade pip setuptools && pip3 install tensorrt_llm
```

Comment:
Could you explain why was the dependency on ZMQ added here? Is it because NIXL is enabled by default? If that's the case, I suggest not making ZMQ a default dependency. Instead, prompt the user with the installation command line the first time they enable dist-serving.

In addition, this kind of example installation command appears more than once in the documentation, so we need to ensure consistency.